### PR TITLE
Vpc documentation

### PR DESCRIPTION
**Pull request by krjackso**

### CODE REVIEW

File: modules/vpc.md [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+Virtual Private Cloud
+---------------------
+
+Each VPC is defined in its own file where the file name is the name of the VPC. These files are located in a [configurable](#configuration) folder. VPCs are JSON objects with the following attributes:
+
+* `cidr-block` - the network range for the VPC in CIDR notation e.g. `"10.0.0.0/16"`
+* `tenancy` - the tenancy of instances launched in the VPC. A value of `"default"` means that instances can be launched with any tenancy and a value of `"dedicated"` means instances will launch as dedicated instances regardless of the tenancy assigned to the instance at launch
+* `dhcp` - a JSON object containing DHCP options for the VPC. Updating DHCP options causes all existing and new instances launched in the VPC to use the new options
+  * `domain-name-servers` - an array of up to 4 IP addresses for domain name servers or `"AmazonProvidedDNS"`
+  * `domain-name` - if using `"AmazonProvidedDNS"` in `us-east-1` specifiy `"ec2.internal"`. If using `"AmazonProvidedDNS"` in another region, specify `{region}.compute.internal` (e.g. `"ap-northesat-1.compute.internal"`). Otherwise, specify a domain name
```
Comment (msiebert) [inline modules/vpc.md:20]: I can't tell you how much I hate how there's always a special case for `us-east-1`

File: modules/vpc.md [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+  ],
+  "tags": {
+    "Name": "vpc-1"
+  }
+}
+{% endhighlight %}
+
+### Route Tables
+
+In Cumulus, route tables are configured in separate files from the VPC config to keep the VPC config simple for higher level changes. Route table configurations are stored in a [configurable](#configuration) route-tables directory. A route table has the following properties:
```
Comment (msiebert) [inline modules/vpc.md]: in a configurable directory for route tables.

File: modules/vpc.md [hunk 3]
```diff
@@ -0,0 +1,272 @@
...
+{% endhighlight %}
+
+For more information see the [IAM module]({{ site.baseurl }}/modules/iam.html)
+
+### Network ACLs
+
+Network ACLs provide an optional layer of security for the instances in your VPC. Each Network ACL configuration is a JSON object with the following properties:
+
+* `inbound` - an array of Network ACL entries defining inbound (ingress) rules for the ACL. Each entry is a JSON object with the following properties:
+  * `rule` - the rule number for the entry. ACL entries are processed in ascneding order by rule number. Valid values are in the range [1, 32766]. AWS recommends leaving large spaces between rule numbers so it is easier to add rules between other rules
```
Comment (msiebert) [inline modules/vpc.md]: ascending

File: modules/vpc.md [hunk 4]
```diff
@@ -0,0 +1,272 @@
...
+* `tags` - a JSON object whose key/value pairs correspond to tag keys and values for the VPC e.g. `{ "TagName": "TagValue" }`. In Cumulus, the `"Name"` tag must be defined on a Network ACL in order to reference it in a [subnet](#subnets) configuration
+
+Here is an example of a Network ACL Configuration:
+
+{% highlight json %}
+{
+  "inbound": [
+    {
+      "rule": 100,
+      "protocol": "ICMP",
```
Comment (msiebert) [inline modules/vpc.md]: Maybe `downcase!` these protocol names?

File: modules/vpc.md [hunk 5]
```diff
@@ -0,0 +1,272 @@
...
+}
+{% endhighlight %}
+
+
+### Subnets
+
+In Cumulus, subnets are configured in separate files from the VPC config to keep the VPC config simple for higher level changes. Subnet configurations are stored in a [configurable](#configuration) subnets directory. A subnet has the following properties:
+
+* `cidr-block` - the network range for the subnet in CIDR notation. Cannot be updated once a subnet is created
+* `availability-zone` - the availability zone for the subnet. Cannot be udpated once a subnet is created
```
Comment (msiebert) [inline modules/vpc.md]: updated

File: modules/vpc.md [hunk 6]
```diff
@@ -0,0 +1,272 @@
...
+  "route-table": "route-table-1",
+  "network-acl": "network-acl-1",
+  "availability-zone": "us-east-1e",
+  "tags": {
+    "Name": "subnet-1"
+  }
+}
+{% endhighlight %}
+
+For more information see the [IAM module]({{ site.baseurl }}/modules/iam.html)
```
Comment (msiebert) [inline modules/vpc.md]: whatever for?

File: modules/vpc.md (hunk 1 above)
Comment (krjackso) [inline modules/vpc.md:20]: At least I didn't have to do anything in code for this one...

**Pull request merged**